packages/custom-elements/src/CustomElementRegistry.js

```diff
@@ -67,9 +67,9 @@ export default class CustomElementRegistry {
 
     /**
      * @private
-     * @type {!Array<!CustomElementDefinition>}
+     * @type {!Array<string>}
      */
-    this._pendingDefinitions = [];
+    this._unflushedLocalNames = [];
```

Comment:
Could the `elementsWithPendingDefinitions` map and `_unflushedLocalNames` just be one thing?